include/BSMPT/ThermalFunctions/ThermalFunctions.h

```diff
@@ -32,7 +36,7 @@ const double C_FermionShift = -0.01560316619;
 const double C_BosonTheta = 9.469230596;
 /**
  * constant shift to the polynomial to make the expansions continuous, see
- * delta_- in Eq. (2.43) in the manual
+ * delta_- in Eq. (2.43) in the manual TODO: Rewrite this
```

Comment:
Here is still a todo

Reply:
It is there by mistake. We ended up not needing to do any further changes to the constants.